### PR TITLE
Clean up man pages to match commands

### PR DESCRIPTION
**Pull request by rhatdan**
Also add podman-commands.sh to compare man pages to commands.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@edsantiago @TomSweeneyRedHat @baude @giuseppe @vrothberg @mheon PTAL

**Comment by edsantiago** [comment]
Differences I'm still seeing:
* `podman --help` still includes `sign` (solution: remove from `commands.go:rootCommands`?)
* `podman container --help` does not list `inspect` (and in fact it doesn't even work).
* `podman image --help` lists the option as `rmi`, man page as `rm`

File: docs/podman-pod.1.md [hunk 1]
```diff
@@ -11,16 +11,24 @@ podman pod is a set of subcommands that manage pods, or groups of containers.
...
+| kill    | [podman-pod-kill(1)](podman-pod-kill.1.md)        | Kill the main process of each container in pod.                                |
+| pause   | [podman-pod-pause(1)](podman-pod-pause.1.md)      | Pause one or more pods.                                                        |
+| ps      | [podman-pod-ps(1)](podman-pod-ps.1.md)            | Prints out information about pods.                                             |
+| restart | [podman-pod-restart(1)](podman-pod-restart.1.md)  | Restart one or mode pods.                                                      |
+| rm      | [podman-pod-rm(1)](podman-pod-rm.1.md)            | Remove one or more pods.                                                       |
+| start   | [podman-pod-start(1)](podman-pod-start.1.md)      | Start one or more pods.                                                        |
+| stats   | [podman-pod-stats(1)](podman-pod-stats.1.md)      | Display live stream resource usage stats for the containers in or more pods.   |                                                     |
```
Comment (mheon) [inline docs/podman-pod.1.md]: Extra | at the end?

Comment (edsantiago) [reply docs/podman-pod.1.md]: Confirmed. Good catch.

**Comment by rhatdan** [comment]
@edsantiago Yes my script catches the issues with the command, this PR is just to get the script in and fix the man pages to be correct.

**Comment by rhatdan** [comment]
With Brents Patch this is what I am seeing now.
```
./hack/podman-commands.sh 
diff -B -u /tmp/podman.cmd /tmp/podman.man
diff -B -u /tmp/podman-image.cmd /tmp/podman-image.man
--- /tmp/podman-image.cmd	2019-02-27 15:49:02.798997841 -0500
+++ /tmp/podman-image.man	2019-02-27 15:49:02.825998023 -0500
@@ -1,14 +1,15 @@
 build
 exists
 history
-images
 import
 inspect
+list
 load
+ls
 prune
 pull
 push
-rmi
+rm
 save
 sign
 tag
diff -B -u /tmp/podman-container.cmd /tmp/podman-container.man
--- /tmp/podman-container.cmd	2019-02-27 15:49:02.848998179 -0500
+++ /tmp/podman-container.man	2019-02-27 15:49:02.872998341 -0500
@@ -7,8 +7,11 @@
 exec
 exists
 export
+inspect
 kill
+list
 logs
+ls
 mount
 pause
 port
@@ -25,5 +28,6 @@
 stop
 top
 umount
+unmount
 unpause
 wait
diff -B -u /tmp/podman-system.cmd /tmp/podman-system.man
diff -B -u /tmp/podman-play.cmd /tmp/podman-play.man
diff -B -u /tmp/podman-generate.cmd /tmp/podman-generate.man
diff -B -u /tmp/podman-pod.cmd /tmp/podman-pod.man
diff -B -u /tmp/podman-volume.cmd /tmp/podman-volume.man
```

File: docs/podman.1.md [hunk 2]
```diff
@@ -167,8 +172,9 @@ the exit codes follow the `chroot` standard, see below:
 | [podman-top(1)](podman-top.1.md)          | Display the running processes of a container.                                  |
 | [podman-umount(1)](podman-umount.1.md)    | Unmount a working container's root filesystem.                                 |
 | [podman-unpause(1)](podman-unpause.1.md)  | Unpause one or more containers.                                                |
-| [podman-version(1)](podman-version.1.md)  | Display the Podman version information.                                        |
-| [podman-volume(1)](podman-volume.1.md)    | Manage Volumes.                                                                 |
+| [podman-varlink(1)](podman-varlink.1.md)  | Display the Podman version information.                                        |
+| [podman-version(1)](podman-version.1.md)  | Runs the varlink backend interface.                                            |
+| [podman-volume(1)](podman-volume.1.md)    | Manage Volumes.                                                                |
 | [podman-wait(1)](podman-wait.1.md)        | Wait on one or more containers to stop and print their exit codes.             |
```
Comment (TomSweeneyRedHat) [inline docs/podman.1.md]: While you're in this file, can you remove the short '-s' option from '--storage-driver' as we no longer support that apparently.

File: docs/podman-pod-stats.1.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+% podman-p	qod-stats "1"
```
Comment (TomSweeneyRedHat) [inline docs/podman-pod-stats.1.md]: s/qod-stats/pod-stats/

File: docs/podman-pod-stats.1.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+## NAME
+podman\-pod\-stats - Display a live stream of resource usage statistics for the containers in one or more pods
+
+## SYNOPSIS
+**podman pod stats** [*options*] [*pod*]
+
+## DESCRIPTION
+Display a live stream of containers in one or more pods resource usage statistics
```
Comment (TomSweeneyRedHat) [inline docs/podman-pod-stats.1.md]: Some lines in this page have ending periods, others do not.  It should be consistent and I'd vote for periods.

Comment (rhatdan) [reply docs/podman-pod-stats.1.md]: Lets do that in a separate PR.

File: docs/podman-pod-stats.1.md (hunk 3 above)
Comment (rhatdan) [reply docs/podman-pod-stats.1.md]: fixed.

File: docs/podman.1.md (hunk 2 above)
Comment (rhatdan) [reply docs/podman.1.md]: Fixed

File: docs/podman-pod.1.md (hunk 1 above)
Comment (rhatdan) [reply docs/podman-pod.1.md]: Fixed

File: docs/podman-pod.1.md [hunk 5]
```diff
@@ -11,16 +11,24 @@ podman pod is a set of subcommands that manage pods, or groups of containers.
...
+| inspect | [podman-pod-inspect(1)](podman-pod-inspect.1.md)  | Displays information describing a pod.                                         |
+| kill    | [podman-pod-kill(1)](podman-pod-kill.1.md)        | Kill the main process of each container in pod.                                |
+| pause   | [podman-pod-pause(1)](podman-pod-pause.1.md)      | Pause one or more pods.                                                        |
+| ps      | [podman-pod-ps(1)](podman-pod-ps.1.md)            | Prints out information about pods.                                             |
+| restart | [podman-pod-restart(1)](podman-pod-restart.1.md)  | Restart one or mode pods.                                                      |
+| rm      | [podman-pod-rm(1)](podman-pod-rm.1.md)            | Remove one or more pods.                                                       |
+| start   | [podman-pod-start(1)](podman-pod-start.1.md)      | Start one or more pods.                                                        |
+| stats   | [podman-pod-stats(1)](podman-pod-stats.1.md)      | Display live stream resource usage stats for the containers in or more pods.   |
```
Comment (TomSweeneyRedHat) [inline docs/podman-pod.1.md]: "in or more pods." I think s/b "in one or more pods."

File: hack/podman-commands.sh [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+./bin/podman pod --help | sed -n -e '0,/Available Commands/d' -e '/^Flags/q;p' | sed '/^$/d' | awk '{ print $1 }' > /tmp/podman-pod.cmd
+man docs/podman-pod.1 | sed -n -Ee '0,/COMMANDS/d'  -e 's/^[[:space:]]*//' -e '/^SEE ALSO/q;p' | grep podman | cut -f1 -d' ' | sed 's/^.//' > /tmp/podman-pod.man
+echo diff -B -u /tmp/podman-pod.cmd /tmp/podman-pod.man
+diff -B -u /tmp/podman-pod.cmd /tmp/podman-pod.man
+
+./bin/podman volume --help | sed -n -e '0,/Available Commands/d' -e '/^Flags/q;p' | sed '/^$/d' | awk '{ print $1 }' > /tmp/podman-volume.cmd
+man docs/podman-volume.1 | sed -n -Ee '0,/COMMANDS/d'  -e 's/^[[:space:]]*//' -e '/^SEE ALSO/q;p' | grep podman | cut -f1 -d' ' | sed 's/^.//' > /tmp/podman-volume.man
+echo diff -B -u /tmp/podman-volume.cmd /tmp/podman-volume.man
+diff -B -u /tmp/podman-volume.cmd /tmp/podman-volume.man
```
Comment (TomSweeneyRedHat) [inline hack/podman-commands.sh]: pretty nifty chunk of sed and awk here....

Comment (rhatdan) [reply hack/podman-commands.sh]: Yes, I am sure @nalind Would have done a better job but it gets the job done.
Want to make this a blocker when you call make validate, to make sure we don't have regressions in the future.

File: docs/podman-pod.1.md (hunk 5 above)
Comment (rhatdan) [reply docs/podman-pod.1.md]: Fixed.

File: hack/podman-commands.sh (hunk 6 above)
Comment (edsantiago) [reply hack/podman-commands.sh]: For a blocker, as a later exercise, would you consider the friendly variation I submitted as a PR to your PR? That way there's less duplication and less hardcoding: if/when podman gains a new subcommand, my variation will pick it up automatically.

Comment (rhatdan) [reply hack/podman-commands.sh]: Sure.  I'll look at it.  My only concern is the script is a little fragile.

**Comment by edsantiago** [comment]
/retest

(looks like a flake.) Last 3 lines of build.log.txt are:
```
2019/02/27 22:51:32 Tagging fedora-install into stable
2019/02/27 22:51:33 Ran for 7m31s
error: could not run steps: could not wait for build: the build centos-install failed after 2s with reason GenericBuildFailed: Generic Build failure - check logs for details.
```

**Pull request merged**